operations/patch_util.go

```diff
@@ -459,6 +459,31 @@ func (p *patchParams) getDescription() string {
 	return description
 }
 
+func (p *patchParams) getModulePath(conf *ClientSettings, module string) (string, error) {
+	modulePath := conf.getModulePath(p.Project, module)
+	if modulePath != "" || p.SkipConfirm {
+		return modulePath, nil
+	}
+
+	modulePath = prompt(fmt.Sprintf("Enter absolute path to module '%s' to include changes (optional):", module))
+	if modulePath == "" {
+		return "", errors.Errorf("no module path given")
+	}
+
+	if !conf.DisableAutoDefaulting {
+		// Verify that the path is correct before auto defaulting
+		if _, err := gitUncommittedChanges(modulePath); err != nil {
+			return "", errors.Wrapf(err, "verifying module '%s''", module)
+		}
+		conf.setModulePath(p.Project, module, modulePath)
+		if err := conf.Write(""); err != nil {
+			grip.Errorf("problem setting module '%s' path in config: %s", module, err.Error())
```

Comment:
nit: remove problem

Reply:
That's actually only for wrapped errors since it bogs down the error to have each iteration of the wrap to say error/problem.